{
  "path": "/documentation/foundation/recoverableerror/2293222-attemptrecovery",
  "type": "Instance Method",
  "name": "attemptRecovery(optionIndex:)",
  "desc": "Attempt to recover from this error when the user selected the option at the given index. Returns true to indicate successful recovery, and false otherwise.Required.",
  "items": [],
  "declaration": "func attemptRecovery(optionIndex recoveryOptionIndex: Int) -> Bool"
}